examples/pytorch/mnist/mnist.py

```diff
@@ -10,141 +10,216 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
-
-WORLD_SIZE = int(os.environ.get('WORLD_SIZE', 1))
+from torch.utils.data import DistributedSampler
```

Comment:
Could we apply the same approach to Katib?

https://github.com/kubeflow/katib/blob/a2f3fcae55d608a850eaea7ff3d25c667d2423e1/examples/v1beta1/trial-images/pytorch-mnist/mnist.py#L29

Reply:
Sure, I will submit separate PR